### PR TITLE
Prints for the pid <-> BLS PK mapping

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added prints for the pid <-> BLS PK mapping

### CODE REVIEW

File: sharding/networksharding/peerShardMapper.go [hunk 1]
```diff
@@ -254,34 +259,37 @@ func (psm *PeerShardMapper) updatePeerIdShardId(pid core.PeerID, shardId uint32)
 	psm.fallbackPidShardCache.HasOrAdd([]byte(pid), shardId, uint32Size)
 }
 
-func (psm *PeerShardMapper) updatePeerIDPublicKey(pid core.PeerID, pk []byte) {
-	//mutUpdatePeerIdPublicKey is used as to consider this function a critical section
+// updatePeerIDPublicKey will update the pis <-> pk mapping, returning true if the pair is a new known pair
```
Comment (bogdan-rosianu) [inline sharding/networksharding/peerShardMapper.go:262]: pid*

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: sharding/networksharding/peerShardMapper.go (hunk 1 above)
Comment (iulianpascalau) [reply sharding/networksharding/peerShardMapper.go]: fixed

**Review by bogdan-rosianu — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

File: sharding/networksharding/peerShardMapper.go [hunk 2]
```diff
@@ -293,38 +301,42 @@ func (psm *PeerShardMapper) updatePeerIDPublicKey(pid core.PeerID, pk []byte) {
 	}
 	psm.pkPeerIdCache.Put(pk, pq, pq.size())
 	psm.peerIdPkCache.Put([]byte(pid), pk, len(pk))
+
+	return isNew
 }
 
-func (psm *PeerShardMapper) removePidAssociation(pid core.PeerID) {
+// removePidAssociation remove the pid association between the pid and public key, returning old public key stored, if existing
```
Comment (raduchis) [inline sharding/networksharding/peerShardMapper.go:308]: removePidAssociation remove**s** ....

**Review by raduchis — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**